project_partner_name/README.rst

```diff
@@ -0,0 +1,49 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=========================
+Project Partner Name
+=========================
+
+This module extends the functionality of Project:
+
+* Makes "Customer" mandatory field.
+* Modifies the name_get adding the customer reference to the project name.
+* Modifies the name_search to search also by customer reference linked to project.
```

Comment:
@elicoidal this PR is only move the module and rename it from elico_odoo , after it merged , Frank will work on these changes.